diff --git a/PKGBUILD b/PKGBUILD
index 66394eb..b7d0bd6 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -80,18 +80,40 @@ package() {
   cd "$pkgname/src/powershell-unix"
 
   mkdir -p "$pkgdir/usr/lib/$pkgname"
-  cp -a "bin/Linux/netcoreapp2.1/linux-x64" "$pkgdir/usr/lib/$pkgname"
-  chmod 755 "$pkgdir/usr/lib/$pkgname/linux-x64/$binaryname"
+  cp -a "bin/." "$pkgdir/usr/lib/$pkgname"
+  cp bin/*.a "$pkgdir/usr/lib/$pkgname"
+  chmod 755 "$pkgdir/usr/lib/$pkgname/$binaryname"
+
+  mkdir -p $pkgdir/usr/lib/powershell/Modules/PackageManagement/1.4.5
+  pushd $pkgdir/usr/lib/powershell/Modules/PackageManagement/1.4.5
+  curl 'https://psg-prod-eastus.azureedge.net/packages/packagemanagement.1.4.5.nupkg' | unzip -
+  mv [Content_Types].xml PSGetModuleInfo.xml
+  popd
+  mkdir -p $pkgdir/usr/lib/powershell/Modules/PowerShellGet/2.2.2
+  pushd $pkgdir/usr/lib/powershell/Modules/PowerShellGet/2.2.2
+  curl 'https://psg-prod-eastus.azureedge.net/packages/powershellget.2.2.2.nupkg' | unzip -
+  mv [Content_Types].xml PSGetModuleInfo.xml
+  popd
+  mkdir -p $pkgdir/usr/lib/powershell/Modules/NuGet/1.3.3
+  pushd $pkgdir/usr/lib/powershell/Modules/NuGet/1.3.3
+  curl 'https://psg-prod-eastus.azureedge.net/packages/nuget.1.3.3.nupkg' | unzip -
+  mv [Content_Types].xml PSGetModuleInfo.xml
+  popd
+  mkdir -p $pkgdir/usr/lib/powershell/Modules/PSReadLine/2.0.0
+  pushd $pkgdir/usr/lib/powershell/Modules/PSReadLine/2.0.0
+  curl 'https://psg-prod-eastus.azureedge.net/packages/psreadline.2.0.0-rc1.nupkg' | unzip -
+  mv [Content_Types].xml PSGetModuleInfo.xml
+  popd
 
   mkdir -p "$pkgdir/usr/share/licenses/$pkgname"
   cp "../../LICENSE.txt" "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
 
   mkdir -p "$pkgdir/usr/bin"
-  ln -s "/usr/lib/$pkgname/linux-x64/$binaryname" "$pkgdir/usr/bin/$binaryname"
+  ln -s "/usr/lib/$pkgname/$binaryname" "$pkgdir/usr/bin/$binaryname"
 
   chmod 644 \
-    "$pkgdir/usr/lib/powershell/linux-x64/libhostfxr.so" \
-    "$pkgdir/usr/lib/powershell/linux-x64/libhostpolicy.so" \
-    "$pkgdir/usr/lib/powershell/linux-x64/en-US/default.help.txt" \
-    "$pkgdir/usr/lib/powershell/linux-x64/Modules/PSDesiredStateConfiguration/PSDesiredStateConfiguration.psm1"
+    $pkgdir/usr/lib/powershell/libhostfxr.so \
+    $pkgdir/usr/lib/powershell/libhostpolicy.so \
+    $pkgdir/usr/lib/powershell/en-US/default.help.txt \
+    $pkgdir/usr/lib/powershell/Modules/PSDesiredStateConfiguration/PSDesiredStateConfiguration.psm1
 }
